### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by phschaad**
- [x] Move backend to a random port
- [x] Add tooltips to editable properties (using the 'desc' attribute provided in its metadata)
- [x] Fix CSS colors for inputs
- [x] Update README and provide screenshots / GIFs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,28 +9,76 @@
...
+of entire SDFGs from scratch.
+
+<!--
+![sdfg-editor-adding-elements-example](images/sdfg_adding_elements.gif)
+-->
+
+## SDFG Optimization / Transformations
+
+With the SDFG Editor, data-centric applications can be optimized with a simple,
+click-based, interactive process using transformations.
```
Comment (tbennun) [inline README.md:38]:
```suggestion
With the SDFG Editor, data-centric applications can be optimized interactively using transformations.
```

File: README.md [hunk 2]
```diff
@@ -9,28 +9,76 @@
...
+
+- A set of applicable performance optimizing transformations are shown in a
```
Comment (tbennun) [inline README.md:40]:
```suggestion
- A set of applicable performance optimizing transformations is shown in a
```

File: src/daceInterface.ts [hunk 3]
```diff
@@ -89,12 +90,31 @@ implements MessageReceiverInterface {
...
+
+    private getRandomPort(callback: CallableFunction) {
+        const rangeMin = 1024;
+        const rangeMax = 65535;
+        const portCandidate = Math.floor(
+            Math.random() * (rangeMax - rangeMin) + rangeMin
+        );
+
+        const tempServer = net.createServer();
+        tempServer.listen(portCandidate, () => {
```
Comment (tbennun) [inline src/daceInterface.ts:103]: why not do it with port 0?

**Review by tbennun — CHANGES_REQUESTED** [review]

File: src/daceInterface.ts (hunk 3 above)
Comment (phschaad) [reply src/daceInterface.ts:103]: Getting the port after using that method is messy (string parsing), but we crucially need it.

**Review by tbennun — APPROVED** [review]

**Pull request merged**